app/views/tasks/_message.html.erb

```diff
@@ -0,0 +1,10 @@
+<% if current_user.id != message.user_id %>
+  <strong> <%= message.user.full_name %> </strong>
+  <small><%= message.created_at.strftime("%H:%M %d.%m.%y") %></small>
+  <p id="collocutors_message_decoration"> <%= message.text %> </p>
+<% else %>
+  <strong class="pull-right"><%= message.user.full_name %></strong>
+  <small>
+  <%= message.created_at.strftime("%H:%M %d.%m.%y") %></small>
```

Comment:
I really can't understand how to do it. I see one line that is equal in both parts. But it is situated between two lines that are unique.